openqaoa/problems/helper_functions.py

```diff
@@ -29,41 +29,34 @@ def convert2serialize(obj):
         return obj
 
 
-def check_kwargs(list_expected_params, list_default_values, **kwargs):
+def convert_binary_to_ising(terms, weights):
     """
-    Checks that the given list of expected parameters can be found in the
-    kwargs given as input. If so, it returns the parameters from kwargs, else
-    it raises an exception.
-
-    Args:
-        list_expected_params: List[str]
-            List of string containing the name of the expected parameters in 
-            kwargs
-        list_default_values: List
-            List containing the deafult values of the expected parameters in 
-            kwargs
-        **kwargs:
-            Keyword arguments where keys are supposed to be the expected params
-
-    Returns:
-        A tuple with the actual expected parameters if they are found in kwargs.
-
-    Raises:
-        ValueError: 
-            If one of the expected arguments is not found in kwargs and its 
-            default value is not specified.
+    Converts the weights from a [0, 1] encoding to an Ising problem [-1, 1] 0 is mapped to +1 and 1 to -1 respectively
+
+    Parameters
+    ----------
+    terms: list[list]
```

Comment:
Is this typing correct. Should its be `list[list]` or `list[str]`

Reply:
I think it's already correct this way, @vishal-ph ?